src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py

```diff
@@ -914,19 +914,8 @@ def body(self):
         extension_name = 'VMAccessForLinux'
         vm_name = 'yugangw8-1'
         resource_group = 'yugangw8'
-        public_key = ('ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC8InHIPLAu6lMc0d+5voyXqigZfT5r6fAM1+FQAi+mkPDdk2hNq1BG0Bwfc88G'
-                      'm7BImw8TS+x2bnZmhCbVnHd6BPCDY7a+cHCSqrQMW89Cv6Vl4ueGOeAWHpJTV9CTLVz4IY1x4HBdkLI2lKIHri9+z7NIdvFk7iOk'
-                      'MVGyez5H1xDbF2szURxgc4I2/o5wycSwX+G8DrtsBvWLmFv9YAPx+VkEHQDjR0WWezOjuo1rDn6MQfiKfqAjPuInwNOg5AIxXAOR'
-                      'esrin2PUlArNtdDH1zlvI4RZi36+tJO7mtm3dJiKs4Sj7G6b1CjIU6aaj27MmKy3arIFChYav9yYM3IT')
         user_name = 'yugangw'
```

Comment:
That makes sense for this one since changing the resource group name would necessitate re-recording.

---
In reply to: [76100883](https://github.com/Azure/azure-cli/pull/737#discussion_r76100883) [](ancestors = 76100883,76078325)